### PR TITLE
Rendering youtube videos via action cable

### PR DESCRIPTION
**Pull request by sergiwez**
**CHECK LIST**
- [ ] specs added
- [ ] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [ ] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [ ] screenshot of rails_best_practices and other tools added
- [ ] screenshot of feature is added if task has UI

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -28,4 +28,6 @@
 //= require react
 //= require react_ujs
 //= require components
-//= require facebook.js.coffee.erb
\ No newline at end of file
+//= require facebook.js.coffee.erb
+//= require cable
+//= require turbolinks
```
Comment (igorkasyanchuk) [inline app/assets/javascripts/application.js]: remove turbolinks

Comment (sergiwez) [reply app/assets/javascripts/application.js]: donee

File: Gemfile [hunk 2]
```diff
@@ -42,6 +43,7 @@ gem 'sentry-raven'
 gem 'sidekiq'
 gem 'simple_form'
 gem 'slim'
+gem 'turbolinks'
```
Comment (igorkasyanchuk) [inline Gemfile]: remove gem

Comment (sergiwez) [reply Gemfile]: +

**Pull request merged**